### DISCUSSION TITLE
听说今天openai关闭了非开放国家和地区的api使用，会影响开源版本的部署使用嘛？在线版呢?

### DISCUSSION THREAD

**Discussion by qnmlgbd250** [labring/FastGPT#1841] [Q&A]
听说今天openai关闭了非开放国家和地区的api使用，会影响开源版本的部署使用嘛？在线版呢?

**Comment by c121914yu** [comment]
> 听说今天openai关闭了非开放国家和地区的api使用，会影响开源版本的部署使用嘛？在线版呢?

在线版是国外的
